Gemfile.lock

```diff
@@ -61,19 +63,18 @@ GEM
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    railties (6.0.3.3)
-      actionpack (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
+    railties (6.1.3)
+      actionpack (= 6.1.3)
+      activesupport (= 6.1.3)
       method_source
       rake (>= 0.8.7)
-      thor (>= 0.20.3, < 2.0)
-    rake (13.0.1)
-    thor (1.0.1)
-    thread_safe (0.3.6)
-    tzinfo (1.2.7)
-      thread_safe (~> 0.1)
-    zeitwerk (2.4.0)
-    zxcvbn-ruby (1.1.0)
+      thor (~> 1.0)
+    rake (13.0.3)
+    thor (1.1.0)
```

Comment:
### [thor](http://whatisthor.com/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.0.1 → 1.1.0

[_[change-log](https://github.com/erikhuda/thor/blob/master/CHANGELOG.md), [source-code](https://github.com/erikhuda/thor/tree/v1.1.0)_]





<details>
<summary>Commits</summary>

A change of **29** commits. See the full changes on [the compare page](https://github.com/erikhuda/thor/compare/v1.0.1...v1.1.0).

These are the first 10 commits:
- (21e1da2) [Don't use ANSI colors when terminal is dumb](https://github.com/erikhuda/thor/commit/21e1da2e8cbc374e2c50a2c2def381c577ee6c2a)
- (6610df3) [Merge pull request #710 from tpope/dumb-terminal](https://github.com/erikhuda/thor/commit/6610df381dd37c685867add35bf7ea71eb9d5651)
- (a8050c8) [Restore colors when TERM is unset](https://github.com/erikhuda/thor/commit/a8050c884879896834fb7f532e866662649d1bd9)
- (2899621) [Merge pull request #711 from tpope/missing-term](https://github.com/erikhuda/thor/commit/2899621975376d622c273eebdaace54bd6c89e4c)
- (97baa6a) [Fix indentation](https://github.com/erikhuda/thor/commit/97baa6a3b8df5a141c7b2e3ad8cd8dfd9b1c70ff)
- (ff37cba) [Configure rubocop correctly](https://github.com/erikhuda/thor/commit/ff37cbaf4620b413aae89fc6e334da58633dc3a0)
- (0222fe5) [Ensure default option/argument is not erroneously aliased](https://github.com/erikhuda/thor/commit/0222fe52ed3803fe3ee0033da5b6faac5ee6299c)
- (e7aad40) [Don't need next: begin/end block is inside the loop already](https://github.com/erikhuda/thor/commit/e7aad400927888e59910c99977aba53827f248d2)
- (2cc6701) [Merge pull request #715 from tsontario/fix_repeatable](https://github.com/erikhuda/thor/commit/2cc67010bbbb5d31b5d2fdcecea068611476a357)
- (5976ee3) [Update required rspec version](https://github.com/erikhuda/thor/commit/5976ee33de19e43db675d5865c25bbad0814b5bd)


</details>